samples/Meeting/src/app/App.tsx

```diff
@@ -33,7 +31,7 @@ interface Credentials {
 }
 
 interface CallArgs {
-  callLocator: GroupLocator | TeamsMeetingLinkLocator;
+  callLocator: GroupLocator | TeamsMeetingLinkLocator | undefined;
```

Comment:
instead of callLocator being able to be undefined, it would be better to do:
```
const [callArgs, setCallArgs] = useState<CallArgs | undefined>(undefined);
```